src/pages/Launchpad/Splits/Form/index.tsx

```diff
@@ -16,14 +16,6 @@ export default function SplitsForm(props: FormProps) {
   const { watch, resetField } = useFormContext<FV>();
   const isCustom = watch("isCustom");
 
-  useEffect(() => {
```

Comment:
this effect runs when user goes back to this Splits page - transfer to `onChange`